scripts/rageshake.go

```diff
@@ -93,12 +94,12 @@ func main() {
 		summary := fmt.Sprintf(
 			"%s\n\nNumber of logs: %d\nVersion: %s\nUser-Agent: %s\n", p.Text, len(p.Logs), p.Version, p.UserAgent,
 		)
-		if err := gzipAndSave([]byte(summary), prefix+".log.gz"); err != nil {
+		if err := gzipAndSave([]byte(summary), prefix, "details.log.gz"); err != nil {
 			respond(500, w)
 			return
 		}
 		for i, log := range p.Logs {
-			if err := gzipAndSave([]byte(log.Lines), fmt.Sprintf("%s-%d.log.gz", prefix, i)); err != nil {
+			if err := gzipAndSave([]byte(log.Lines), prefix, fmt.Sprintf("logs-%d.log.gz", i)); err != nil {
```

Comment:
> What's the reason for saving each one as a separate file rather than concatenating them all together (perhaps even with a separator)? Do the logs have timestamps?

The logs have timestamps. We cannot concatenate all the files together as that breaks causality. The number of files corresponds to the number of *sessions* where a session is effectively a browser tab. vdh wants to be able to see logs for each browser tab side-by-side rather than muxxed together. A session's position in the list of files is governed by the most-recent log timestamp.

> Also I would suggest some level of slightly deeper nesting, like at least YYYY-MM-DD/HHMMSS/ otherwise you're going to find that before long, the directory kills your terminal if you type ls.

Good idea :+1: 

> (and you already know my view on using timestamps as IDs, and not particularly fine-grained timestamps at that)

Well given we are now representing timestamps as directories, I don't mind having a monotonically increasing integer as part of the filename, though it might be weird to see 2 bug reports in the same directory?